runtime/ajuna/src/xcm_config.rs

```diff
@@ -425,11 +428,32 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type VersionWrapper = PolkadotXcm;
 	// Enqueue XCMP messages from siblings for later processing.
 	type XcmpQueue = TransformOrigin<MessageQueue, AggregateMessageOrigin, ParaId, ParaIdToSibling>;
-	type MaxInboundSuspended = sp_core::ConstU32<1_000>;
+	type MaxInboundSuspended = ConstU32<1_000>;
+	type MaxActiveOutboundChannels = ConstU32<128>;
+	// Most on-chain HRMP channels are configured to use 102400 bytes of max message size, so we
+	// need to set the page size larger than that until we reduce the channel size on-chain.
+	type MaxPageSize = ConstU32<{ 103 * 1024 }>;
 	type ControllerOrigin = EnsureRoot<AccountId>;
 	type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
-	type WeightInfo = cumulus_pallet_xcmp_queue::weights::SubstrateWeight<Runtime>;
 	type PriceForSiblingDelivery = NoPriceForMessageDelivery<ParaId>;
+	type WeightInfo = cumulus_pallet_xcmp_queue::weights::SubstrateWeight<Runtime>;
+}
+
+pub struct AjunaActiveXCMChannels;
+
+/// Polkadot AssetHub parachain identifier
+pub const ASSET_HUB_PARA_ID: ParaId = ParaId::new(1000);
+/// Polkadot HydraDX parachain identifier
+pub const HYDRA_DX_PARA_ID: ParaId = ParaId::new(2034);
+
+impl ListChannelInfos for AjunaActiveXCMChannels {
+	fn outgoing_channels() -> Vec<ParaId> {
+		vec![ASSET_HUB_PARA_ID, HYDRA_DX_PARA_ID]
+	}
+}
+
+impl cumulus_pallet_xcmp_queue::migration::v5::V5Config for Runtime {
+	type ChannelList = AjunaActiveXCMChannels;
 }
```

Comment:
Unfortunately, I realized that we can't do it like this, as the actual runtime migration on Paseo will fail, not only the try runtime one.


I will check the outgoing connections manually myself for sanity, and then write a custom migration that is a no-op and simply updates the storage version of the xcm queue.